sdk/sdk/Models/dao/OBAModelDAOUserPreferencesImpl.m

```diff
@@ -243,8 +235,6 @@ - (void) writeSetRegionAutomatically:(BOOL)setRegionAutomatically {
     NSUserDefaults * user = [NSUserDefaults standardUserDefaults];
     NSNumber * v = @(setRegionAutomatically);
     [user setObject:v forKey:kSetRegionAutomaticallyKey];
-
-    [[GAI sharedInstance].defaultTracker set:[GAIFields customDimensionForIndex:2] value:(setRegionAutomatically ? @"YES" : @"NO")];
```

Comment:
Makes sense my concern is just people calling `self.modelDao writeSetRegionAutomatically`, etc. rather than the correct function that also sets the analytic event.